tests/integration/utils.py

```diff
@@ -93,6 +93,29 @@ def wait_for_complete_request(self, response):
 
         return response
 
+    def fetch_all_requests(self, request_json=None):
```

Comment:
Could you rename `request_json` to `query_params` so it's clear what it's used for?
```suggestion
    def fetch_all_requests(self, query_params=None):
```

Reply:
done, thank you! 

I changed it in assertions too.